.github/workflows/check-manifests.yml

```diff
@@ -26,12 +26,12 @@ jobs:
     - name: Check x86_64 manifests
       run: |
         pushd toolkit
-        make check-x86_64-manifests
+        ./scripts/toolchain/check_manifests.sh -a "x86_64"
```

Comment:
I feel like we want to have the pipelines use the tools the same way a user does. Is there a reason we want to change the users' behavior here?

Reply:
I think it's a great step in empowering the user to call scripts/ tools directly without having to invoke make. Would help us redesign/ get rid of make in the future more easily.